src/lib/core/a11y/list-key-manager.ts

```diff
@@ -32,10 +32,18 @@ export class ListKeyManager {
     return this;
   }
 
-  /** Sets the focus of the list to the item at the index specified. */
-  setFocus(index: number): void {
+  /**
+   * Sets the focus of the list to the item at the index specified.
+   *
+   * @param index The index of the item to be focused.
+   * @param focusElement Whether or not to focus the element as well. Defaults to `true`.
+   */
+  setFocus(index: number, focusElement = true): void {
```

Comment:
Should this just be `set focusedItemIndex()`?

I REALLY don't like doing `manager.focusedItemIndex = ...`, so I would prefer your suggestion of `updateFocusedItemIndex(index)`, but I was putting it directly below the getter and it felt a bit weird.

Thoughts?